src/vivarium_census_prl_synth_pop/results_processing/addresses.py

```diff
@@ -145,3 +148,29 @@ def get_household_address_map(
         address_map[obs_column] = address_data[obs_column]
 
     return address_map
+
+
+def get_city_map(
+    column_name: str,
+    obs_data: pd.DataFrame,
+    artifact: Artifact,
+    randomness: RandomnessStream,
+) -> Dict[str, pd.Series]:
+
+    # Load addresses data from artifact
+    addresses = artifact.load(data_keys.SYNTHETIC_DATA.ADDRESSES).reset_index()
+    # Get observer data to map
+    output_cols = [column_name, "state"]
+    city_data = obs_data.reset_index()[output_cols].drop_duplicates().set_index("address_id")
+
+    for state in city_data["state"].str.lower().unique():
+        cities = vectorized_choice(
+            options=addresses.loc[addresses["Province"] == state, "Municipality"],
```

Comment:
Does each city in a given state have an equal change of being chosen?

Reply:
That looks like it. The documentation link here doesn't seem to imply they need to match puma https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#id34

@albrja can you please confirm?

Reply:
See [my reply below](https://github.com/ihmeuw/vivarium_census_prl_synth_pop/pull/213#discussion_r1107634138).